pkg/controller/controller.go

```diff
@@ -1714,3 +1719,7 @@ func checkFinalizer(obj metav1.Object, finalizer string) bool {
 	}
 	return false
 }
+
+func markAsMigrated(parent context.Context, hasMigrated bool) context.Context {
```

Comment:
@Jiawei0227 It would probably be better to move this to the csi-lib-utils code because it wil be used across all the sidecars. What do you say?

Reply:
Well. Yeah, I agree. But this function seems too small and I think it is okay to move around. So let's just keep it here maybe